renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -386,8 +386,15 @@ ReplayStatus GLInitParams::Serialise()
   return ReplayStatus::Succeeded;
 }
 
-void WrappedOpenGL::GetGLExtensions()
+const vector<string> &WrappedOpenGL::GetGLExtensions()
 {
+  static vector<string> globalExts;
```

Comment:
This isn't thread-safe if a second context gets activated and comes in here while the first context is still filling the array.

Since we have separate functions anyway, can't we just split the previous `globalExts` member into `m_SupportedGLExtensions` and `m_SupportedGLESExtensions` and initialise them both in the `WrappedOpenGL` constructor. Then later do:

```cpp
const vector<string> &globalExts = IsGLES ? m_SupportedGLESExtensions : m_SupportedGLExtensions;
```

The cost of initialising both arrays is negligible.